test/runtests.jl

```diff
@@ -1,2 +1,6 @@
-# write your own tests here
-include("singlethreaded.jl")
+using ReadWriteLocks
+using Test
+
+@testset "ReadWriteLocks" begin
```

Comment:
Now that Julia 1.2 and 1.3 exist, we should add tests using actual threads. We can talk about that on Slack.

Reply:
in case we do this in a follow-up PR, opened https://github.com/invenia/ReadWriteLocks.jl/issues/6